bears/js/JSStandardBear.py

```diff
@@ -16,13 +18,53 @@ class JSStandardBear:
     """
 
     LANGUAGES = {'JavaScript', 'JSX'}
-    REQUIREMENTS = {NpmRequirement('standard', '10')}
+    REQUIREMENTS = {NpmRequirement('standard', '10'),
+                    NpmRequirement('typescript-eslint-parser', '12.0.0'),
+                    NpmRequirement('eslint-plugin-typescript', '0.8.1'),
+                    NpmRequirement('eslint-plugin-markdown', '1.0.0-beta.6'),
+                    NpmRequirement('eslint-plugin-html', '3.2.2')}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax'}
     CAN_FIX = {'Formatting'}
     SEE_MORE = 'https://standardjs.com/rules.html'
 
-    def create_arguments(self, filename, file, config_file):
-        return (filename, '--verbose')
+    def create_arguments(self, filename, file, config_file,
+                         language: language = language('JavaScript'),
+                         global_vars: typed_list(str) = (),
+                         eslint_env: typed_list(str) = (),
+                         ):
+        """
+        :param language:
```

Comment:
all of these plugins, and all of these options, all apply to `eslint`.  It is going to be wierd if JSStandard bear is wildly different from ESLint, e.g. doesnt use `babel-eslint`.

Soooo ... move allllll of this stuff to `ESLintBear`.

Very likely, all of your tests also are almost applicable for `ESLintBear`.

Dont want to write extra tests?  No worries.

Make StandardBear inherit from ESLintBear.
All of your current tests for StandardBear will cover the lines of ESLintBear that are moved there.

`standard` ignores the `--stdin` and `--stdin-filename` args which ESLintBear users.  Create an upstream bug about those -- we want those.  json is yummier than plain text.

But we can workaround it atm:

`JSStandardBear.create_arguments` would call `super` `ESLintBear.create_arguments` , and then add the filename to the end of the args.

`JSStandardBear.process_output` would skip `ESLint.process_output` and call `Bear.process_output`

It also ignores `--no-color` `--no-ignore` but meh .. we dont need those anyway.

`ASCIINEMA_URL` needs to be explicitly set to `None` to override `ESLint`.

Reply:
`--stdin` and `--stdin-filename` are already supported I believe. I tested by running `cat test.js | standard --stdin --stdin-filename=test.js`. The documentation links to [standard-json](https://www.npmjs.com/package/standard-json) package for formatting to json output.

Otherwise I generally agree with your idea seeing that standard is only using rules from eslint. I should be able to move my code from here directly to `EslintBear`.

Reply:
Assigned to you.  Please just do it in this PR.

And as *everything* in this PR can be put in `ESLintBear`, there will be another commit in this PR for adapting JSStandardBear to inheriting from ESLintBear.  I want to see those two together, so we get it right.

Otherwise we'll change ESLintBear incorrectly, and wont be able to build JSStandardBear on top.